src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/PublicationServiceTests.cs

```diff
@@ -222,6 +222,11 @@ await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId)
 
                 Assert.Equal(publication.Id, publicationViewModel.Id);
                 Assert.True(publicationViewModel.IsSuperseded);
+                
+                Assert.NotNull(publicationViewModel.SupersededBy);
+                Assert.Equal(publication.SupersededBy.Id, publicationViewModel.SupersededBy?.Id);
+                Assert.Equal(publication.SupersededBy.Title, publicationViewModel.SupersededBy?.Title);
+                Assert.Equal(publication.SupersededBy.Slug, publicationViewModel.SupersededBy?.Slug);
```

Comment:
I think this should be

```
                Assert.NotNull(publicationViewModel.SupersededBy);
                Assert.Equal(publication.SupersededBy.Id, publicationViewModel.SupersededBy!.Id);
                Assert.Equal(publication.SupersededBy.Title, publicationViewModel.SupersededBy.Title);
                Assert.Equal(publication.SupersededBy.Slug, publicationViewModel.SupersededBy.Slug);
```

where you only need one null-forgiving `!` operator rather than the three null-conditional operators.

I think you should also give the `SupersededBy` publication on L174 a `Title` and a `Slug` otherwise you'll see we are just comparing the default empty string values here if you debug.

Reply:
Looks like this hasn't been changed?

Reply:
I think it's up to date now with the expected changes. I'm going to approve. 🙂